### PR TITLE
add step sequence feature

### PR DESCRIPTION
**Pull request by marcinstachowicz**

### CODE REVIEW

File: cypress/locators/step-sequence/locators.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// component preview locators
+export const STEP_SEQUENCE = '#story-root > ol';
```
Comment (DlgSHi) [inline cypress/locators/step-sequence/locators.js]: should we start from story_root ?

If so, maybe we should import storyRoot() ? as U used below

File: cypress/support/step-definitions/step-sequence-steps.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const ARIA_LABEL = 'Step 1 of 5';
+
+Then('orientation is set to {string}', (orientation) => {
+  stepSequence().should('have.attr', 'orientation', orientation);
+  stepSequenceElement().should('have.attr', 'orientation', orientation);
+});
+
+Then('indicator {string} is set', (indicator) => {
+  cy.wait(100); // required because iframe content is changed
```
Comment (DlgSHi) [inline cypress/support/step-definitions/step-sequence-steps.js]: I was wondering, could we figure out something to use another wait, not limited in time -> cy.wait('@getStorybook') ?

File: cypress/locators/step-sequence/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {
+  STEP_SEQUENCE, STEP_SEQUENCE_ELEMENT, STEP_SEQUENCE_ITEM_INDICATOR, STEP_SEQUENCE_ITEM,
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/locators/step-sequence/index.js]: eslint not indicates separation

File: cypress/support/step-definitions/step-sequence-steps.js (hunk 2 above)
Comment (marcinstachowicz) [reply cypress/support/step-definitions/step-sequence-steps.js]: storybook element exists all time, only value or content of that element is changed, can be add to do that for maintenance task after finish faze1

File: cypress/integration/stepSequenceItem.feature [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      | ariaLabel   |
+      | Sample text |
+      | 1234567890  |
+      | áéíóú¿¡üñ   |
+      | ÄÖÜßäöüß    |
+
+  @positive
+  Scenario Outline: I set children to <children>
+    When I set children to "<children>"
+    Then children "1 <children>" is set
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/integration/stepSequenceItem.feature]: fixed

File: cypress/integration/stepSequenceItem.feature [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      | status     | color               |
+      | complete   | rgb(0, 163, 118)    |
+      | current    | rgba(0, 0, 0, 0.9)  |
+      | incomplete | rgba(0, 0, 0, 0.55) |
+
+  @positive
+  Scenario Outline: I set hiddenCompleteLabel to <hiddenCompleteLabel>
+    When I set hiddenCompleteLabel to "<hiddenCompleteLabel>"
+      And I select status to "complete"
+    Then hidden label "<hiddenCompleteLabel>" is set
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/integration/stepSequenceItem.feature]: fixed

File: cypress/integration/stepSequenceItem.feature [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+      | Sample text         |
+      | 1234567890          |
+      | áéíóú¿¡üñ           |
+      | ÄÖÜßäöüß            |
+
+  @positive
+  Scenario Outline: I set hiddenCurrentLabel to <hiddenCurrentLabel>
+    When I set hiddenCurrentLabel to "<hiddenCurrentLabel>"
+      And I select status to "current"
+    Then hidden label "<hiddenCurrentLabel>" is set
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/integration/stepSequenceItem.feature]: fixed

File: cypress/integration/stepSequenceItem.feature [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      | hiddenCurrentLabel |
+      | Sample text        |
+      | 1234567890         |
+      | áéíóú¿¡üñ          |
+      | ÄÖÜßäöüß           |
+
+  @positive
+  Scenario Outline: I set ariaLabel to <ariaLabel>
+    When I set ariaLabel to "<ariaLabel>"
+    Then ariaLabel "<ariaLabel>" is set
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/integration/stepSequenceItem.feature]: fixed

File: cypress/locators/step-sequence/locators.js (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/locators/step-sequence/locators.js]: I isolate `#story-root` to local const becasue is not recommended to share locators between components

**Pull request merged**